src/web/components/MainMedia.tsx

```diff
@@ -106,6 +109,22 @@ function renderElement(
 					/>
 				</div>
 			);
+		case 'model.dotcomrendering.pageElements.VideoYoutubeBlockElement':
+			return (
+				<Figure role={element.role}>
```

Comment:
Not sure we want to use Figure in main media

Reply:
- Frontend wraps main media in this case in `figure`
- We use `figurecaption` for the caption, which semantically feels like it should belong in `figure`

For me, this raises the question should we use figure for `YoutubeBlockElement` in main Media as well?

Reply:
We use `Figure` (capital F) for spacing, which does not apply here. The semantics of using a `figure` element is perhaps a separate question